src/etcd/etcd_utils.h

```diff
@@ -0,0 +1,20 @@
+
+#ifndef CREDIS_UTILS_H
+#define CREDIS_UTILS_H
+
+#include <vector>
+#include "etcd3/include/etcd3.h"
```

Comment:
this include is not needed in this header - best to move it to impl file (if needed)